flow/connectors/postgres/cdc.go

```diff
@@ -501,6 +501,7 @@ func PullCdcRecords[Items model.Items](
 					// tableName here is destination tableName.
 					// should be ideally sourceTableName as we are in PullRecords.
 					// will change in future
+					// TODO: replident is cached here, should not cache since it can change
```

Comment:
what do you mean by cached? is there no relation message for this?

Reply:
it's populated from SetupFlow, and I'm not sure if we get this as a relation message